app/services.js

```diff
@@ -1,7 +1,7 @@
 // Services
 
 // Boot service
-airTrafficControlApp.service('systemBootService', ['$location', '$q', '$timeout', function($location, $q, $timeout){
+airTrafficControlApp.service('systemBootService', ['$location', function($location){
     

```

Comment:
Please use the angular factory method to create your service; again do things the angular way, the way the tutorial shows you how to create a service.
Also use the module pattern, return an object with the functions you want available.